test/hooks.js

```diff
@@ -257,12 +257,12 @@ test('shared context', function (t) {
 	var runner = new Runner();
 
 	runner.before(function (a) {
-		a.is(a.context, undefined);
-		a.context = {arr: []};
+		a.is(a.context, null);
+		// a.context = {arr: []};
```

Comment:
Setting `t.context` in `before` or `after` will now throw an error. Previously we silently let them set it, but it was not passed to future tests.